src/pages/GetInvolved/Club.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { withRouter, Link } from "react-router-dom";
+import Layout from "components/Layout/Layout";
+
+const Club = ({ match }) => {
+  let clubDataAll = useSelector((state) => state.clubs);
+  let currentClub = `${match.params.clubId}`;
+  let clubData = useSelector((state) =>
+    clubDataAll.find((object) => object.slug === currentClub)
+  );
+
+  if (!clubData) {
+    return null;
+  } else {
+    const createMarkup = () => {
+      return { __html: clubData.acf.club.club_description };
+    };
+    return (
+      <Layout>
+        <div className="club__container">
+          <p className="club__nav">
+            <Link className="club__nav--item" to="/">
+              Home
+            </Link>{" "}
+            /{" "}
+            <Link className="club__nav--item" to="/clubs">
+              Join a HUOA Club
+            </Link>{" "}
+            / {clubData.acf.seo.title}
+          </p>
+          <br />
+          <div className="club__header">
+            {!clubData.acf.club.club_image ? null : (
+              <img
+                className="club__header--image"
+                src={`${clubData.acf.club.club_image}`}
+                alt="club logo"
+              />
+            )}
+            <div className="club__header--title">{clubData.acf.seo.title}</div>
+          </div>
+          {!clubData.acf.club.club_description ? null : (
+            <div
+              classname="club__description"
+              dangerouslySetInnerHTML={createMarkup()}
+            />
+          )}
+
+          {!clubData.acf.club.club_contact.address &&
+          !clubData.acf.club.club_contact.email &&
+          !clubData.acf.club.club_contact.phone ? null : (
+            <div className="club__contact">
+              <div className="club__contact--title">Contact Information</div>
+              <div className="club__contact--address">
+                {!clubData.acf.club.club_contact.address ? null : (
+                  <div>{clubData.acf.club.club_contact.address}</div>
+                )}
+              </div>
+              <div className="club__contact--email">
+                {!clubData.acf.club.club_contact.email ? null : (
+                  <div>{clubData.acf.club.club_contact.email}</div>
+                )}
+              </div>
+              <div className="club__contact--phone">
+                {!clubData.acf.club.club_contact.phone ? null : (
+                  <div>P: {clubData.acf.club.club_contact.phone}</div>
+                )}
+              </div>
```

Comment:
If these arent defined we shouldnt render an empty div instead lets wrap the whole in the conditional
```suggestion
              {!clubData.acf.club.club_contact.phone ? null : (
                <div className="club__contact--phone">
                  <div>P: {clubData.acf.club.club_contact.phone}</div>
                </div>
              )}
```

Reply:
if you wanted to make this a bit more short hand:

```
              {!!clubData.acf.club.club_contact.phone  && (
                <div className="club__contact--phone">
                  <div>P: {clubData.acf.club.club_contact.phone}</div>
                </div>
              )}
```

The double `!!` is used in js to check for bool value so if phone is undefined it returns false and wont render this section if true it will